docs/running-through-cf-tunnels.md

```diff
@@ -0,0 +1,113 @@
+# Running a Local Dashboard through Cloudflare Tunnels
+
+> ### Note
```

Comment:
This is a nice touch and for sure we should give kudos/reference to where this came from but would probably place this at the end of the page.

Reply:
that's true, will change